### ISSUE TITLE
Define (and document) a common server id like "central" as the default repository

### ISSUE THREAD

**Issue by laeubi** [eclipse/tycho#805]
Currently if one has a repository in the target it is quite cumbersome to define password/proxy settings:

1. one needs to assign an id
2. configure for each id the same settings

If there are general settings (e.g. company nexus mirror requiring auth) it might be good to simply assume if no id is given we search for a fixed id (e.g. p2) like maven "central" id

**Comment by mickaelistria** [comment]
I'm not what is it you're calling the "default" repsitory?
Not that proxy/mirrors are already supported for p2 repos in Tycho using https://wiki.eclipse.org/Tycho/Target_Platform/Authentication_and_Mirrors , which says that if no id is specified, then the URL can be used as id in settings.xml.

**Comment by laeubi** [comment]
Yes but if I want to define a prox/auth for *all* urls then there is no way of doing so, that means I need to configure for each URL the same (what could become cumbersome).

**Comment by mickaelistria** [comment]
I believe proxy settings applies to all connection independently of the server.
For auth, I don't get how 1 auth could be used for all URLs, however, those may match some host or even host name regexp. But I'm not sure it's a Tycho-specific issue. Doesn't Maven already has capabilities to cover that case? IMO we should first have Maven providing a solution for a "shared" auth before building one in Tycho.
